test/tests/functional/pbs_qmgr.py

```diff
@@ -37,10 +37,10 @@
 # trademark licensing policies.
 
 import os
-from string import ascii_letters
 from random import choice
```

Comment:
In my previous comment I meant to point 'from random import choice' as the second redundant import :)